src/Api/AdminConsole/Controllers/PoliciesController.cs

```diff
@@ -85,6 +85,23 @@ public async Task<PolicyDetailResponseModel> Get(Guid orgId, int type)
         return new PolicyDetailResponseModel(policy);
     }
 
+    [HttpGet("{type}/policy-status")]
+    public async Task<bool> GetPolicyStatusAsync(Guid orgId, PolicyType type)
+    {
+        var user = await _userService.GetUserByPrincipalAsync(User);
+        if (user == null)
+        {
+            throw new UnauthorizedAccessException();
+        }
```

Comment:
This authorization check is inadequate. This allows any logged in user to access the policy for any organization.

However, I'm not sure we need a new endpoint here at all. Please see my comment on your clients PR: https://github.com/bitwarden/clients/pull/12056#discussion_r1851238497

Reply:
You are right, I dont need a new endpoint, I will close the pr. Thank you